print/fonttools-utils/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+name                fonttools-utils
+github.setup        twardoch fonttools-utils 43062f972f5e7d95181ea57bfeb68b00812cf357
+version             20170111
+description         Font-related tools based on FontTools
+long_description    Font-related tools that use the Python FontTools \
+                    package: MacOSXSystemFontReplacer replaces UI fonts\; \
+                    pyftfeatfreeze "freezes" OpenType features into a font
+platforms           darwin
+categories          print
+license             Apache-2.0
+maintainers         {@amake madlon-kay.com:aaron+macports} openmaintainer
+
+checksums           rmd160  43c26e7810bee3d07f9fdf3e67158892b19384e3 \
+                    sha256  5b4943ec68e937f8da3119edc11e080e7c4a8ff5cdf61378fdc327a9e3b7aa9f
+
+python.default_version 27
+
+depends_run         port:fonttools
+
+build.cmd           true
+
+destroot {
+    set docdir ${prefix}/share/doc/${name}
+
+    foreach tool {mac-os-x-system-font-replacer pyftfeatfreeze} {
+        set tool_dir ${docdir}/${tool}
+        xinstall -d ${destroot}${tool_dir}
+        xinstall -m 0644 -W ${worksrcpath}/${tool} \
+            AUTHORS CONTRIBUTORS LICENSE README.md ${destroot}${tool_dir}
+        xinstall -m 0755 [glob ${worksrcpath}/${tool}/*.py] ${destroot}${prefix}/bin
```

Comment:
These scripts use `#!/usr/bin/env python` as interpreter which should be changed to `#!${python.bin}` with a reinplace. Otherwise there is no guarantee that `python` is actually from python27 which is the only one with the fonttools module.